### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by amolmanjure** [leonidlm/packer-builder-softlayer#17]
I tried installing the builder as per the instructions in the README but the build fails. Here is my output.

```
Amols-iMac:packer-builder-softlayer amol$ echo $GOPATH
/Users/amol/Documents/Development/go
Amols-iMac:packer-builder-softlayer amol$ pwd
/Users/amol/Documents/Development/go/src/github.com/leonidlm/packer-builder-softlayer
Amols-iMac:packer-builder-softlayer amol$ go build -o /usr/local/packer/packer-builder-softlayer main.go
# github.com/leonidlm/packer-builder-softlayer/builder/softlayer
builder/softlayer/builder.go:45: undefined: packer.ConfigTemplate
Amols-iMac:packer-builder-softlayer amol$ echo $?
2
Amols-iMac:packer-builder-softlayer amol$ go test ./...
# github.com/leonidlm/packer-builder-softlayer/builder/softlayer
builder/softlayer/builder.go:45: undefined: packer.ConfigTemplate
# github.com/leonidlm/packer-builder-softlayer/builder/softlayer
builder/softlayer/builder.go:45: undefined: packer.ConfigTemplate
```
I tried it with the Vagrantbox in case it is an environment issue but see the same error.

````
Amols-iMac:packer-builder-softlayer amol$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'chef/ubuntu-12.04' is up to date...
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Mounting shared folders...
    default: /opt/gopath/src/github.com/leonidlm/packer-builder-softlayer => /Users/amol/Documents/Development/go/src/github.com/leonidlm/packer-builder-softlayer
==> default: Machine already provisioned. Run `vagrant provision` or use the `--provision`
==> default: to force provisioning. Provisioners marked to run always will still run.

Amols-iMac:packer-builder-softlayer amol$ vagrant ssh
Welcome to Ubuntu 12.04.4 LTS (GNU/Linux 3.11.0-15-generic x86_64)

 * Documentation:  https://help.ubuntu.com/
New release '14.04.2 LTS' available.
Run 'do-release-upgrade' to upgrade to it.

Last login: Fri Jun 12 12:25:51 2015 from 10.0.2.2
vagrant@vagrant:~$ cd /opt/gopath/src/github.com/leonidlm/packer-builder-softlayer
vagrant@vagrant:/opt/gopath/src/github.com/leonidlm/packer-builder-softlayer$ go build -o /usr/local/packer/packer-builder-softlayer main.go
# github.com/leonidlm/packer-builder-softlayer/builder/softlayer
builder/softlayer/builder.go:45: undefined: packer.ConfigTemplate
vagrant@vagrant:/opt/gopath/src/github.com/leonidlm/packer-builder-softlayer$ echo $?
2
```

**Comment by cmluciano** [comment]
Yep this looks like a change in packer itself https://github.com/mitchellh/packer/commit/1d3a4d6aa2b2c969bb52f00e27113d21a28ad793

**Comment by leonidlm** [comment]
@amolmanjure @cmluciano guys I probably won't have the time to fix this during the upcoming week, however if one of you would like to submit a patch I will make sure to review and merge it asap.

**Comment by cmluciano** [comment]
Cool, I'll try to look at it this week.

**Comment by cmluciano** [comment]
I'm going to try to fix work on this when I can, but that might take a while since I'm not too familiar with go-lang. An immediate unblocker is just to compile against version 0.6.1 of packer.

**Comment by cmluciano** [comment]
Should be fixed with aaee0561423ff696e3f516895a9ef671b6d2afd6
